Doc/whatsnew/3.7.rst

```diff
@@ -232,6 +232,12 @@ New function :func:`os.register_at_fork` allows registering Python callbacks
 to be executed on a process fork.  (Contributed by Antoine Pitrou in
 :issue:`16500`.)
 
+struct
+------
+
+The :attr:`struct.Struct.format` type is now :class:`str` instead of
```

Comment:
Actually I'm not sure that mentioning this change here is needed. This is not a new *feature*.